File: app/src/main/java/com/yhjoo/dochef/activities/RecipeMakeActivity.java
package com.yhjoo.dochef.activities;

import android.os.Bundle;

import com.yhjoo.dochef.databinding.ARecipemakeBinding;

public class RecipeMakeActivity extends BaseActivity {
    enum MODE {WRITE, REVISE}

    ARecipemakeBinding binding;

    /*
        TODO
        모든 기능
    */

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ARecipemakeBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());

        setSupportActionBar(binding.recipemakeToolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
    }
}

File: app/src/main/java/com/yhjoo/dochef/ui/activities/RecipeMakeActivity.kt
package com.yhjoo.dochef.ui.activities

import android.os.Bundle
import com.yhjoo.dochef.databinding.RecipemakeActivityBinding

class RecipeMakeActivity : BaseActivity() {
    /*
        TODO
        모든 기능
    */

    object MODE {
        const val WRITE = 0
        const val REVISE = 1
    }

    private val binding: RecipemakeActivityBinding by lazy {
        RecipemakeActivityBinding.inflate(
            layoutInflater
        )
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(binding.root)
        setSupportActionBar(binding.recipemakeToolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }
}